shared/django_apps/dummy_settings.py

```diff
@@ -68,6 +94,12 @@
 
 AUTH_PASSWORD_VALIDATORS = []
 
+# See https://django-postgres-extra.readthedocs.io/en/master/settings.html
+POSTGRES_EXTRA_DB_BACKEND_BASE: "django_prometheus.db.backends.postgresql"
```

Comment:
So we didn't have the engine set to `django_prometheus` like api did here, https://github.com/codecov/codecov-api/pull/414/files#diff-82bf6dbe04b3ff913e6d14636041575bd98d8363d424e0168e72bb8ab7c5e700L201, but we want to keep the DB settings aligned don't we?